modules/metastorage/src/test/java/org/apache/ignite/internal/metastorage/server/RocksDbCompactionKeyValueStorageTest.java

```diff
@@ -17,13 +17,29 @@
 
 package org.apache.ignite.internal.metastorage.server;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
 import org.apache.ignite.internal.failure.NoOpFailureManager;
 import org.apache.ignite.internal.metastorage.server.persistence.RocksDbKeyValueStorage;
+import org.junit.jupiter.api.Test;
 
 /** Compaction test for the RocksDB implementation of {@link KeyValueStorage}. */
 public class RocksDbCompactionKeyValueStorageTest extends AbstractCompactionKeyValueStorageTest {
     @Override
     public KeyValueStorage createStorage() {
         return new RocksDbKeyValueStorage("test", workDir.resolve("storage"), new NoOpFailureManager());
     }
+
+    @Test
+    void checksumsAreRemovedForCompactedRevisions() {
```

Comment:
I would suggest adding a test for the compaction case to `org.apache.ignite.internal.metastorage.server.AbstractCompactionKeyValueStorageTest`.

Reply:
As we only support checksums in the RocksDB-based storage (and it makes no sense to support checksums in the in-mem storage), it is natural to only have these tests here, not in the base class

Reply:
Ok